pkgs/tools/misc/pipreqs/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, pythonPackages }:
```

Comment:
Can we use `python3Packages`?

Reply:
Yeah but seems when packaging using `python3Packages`. the binary fails for python code written in python2 (tested the binary on,`nixops` ) fails when it find `print "dsqdqs"`.